--- absl-py-0.9.0/setup.py~	2019-12-17 23:46:45.000000000 +0100
+++ absl-py-0.9.0/setup.py	2022-04-09 10:21:04.611211528 +0200
@@ -29,34 +29,10 @@
   use_setuptools()
   import setuptools
 
-py_version = platform.python_version_tuple()
-if py_version < ('2', '7') or py_version[0] == '3' and py_version < ('3', '4'):
-  raise RuntimeError('Python version 2.7 or 3.4+ is required.')
-
 INSTALL_REQUIRES = [
     'six',
 ]
 
-setuptools_version = tuple(
-    int(x) for x in setuptools.__version__.split('.')[:2])
-
-# A variety of environments have very, very old versions of setuptools that
-# don't support the environment markers ("foo; python_version < X"). Since
-# we're using sdist, this setup.py gets run directly when installing, so
-# we can just manually do the dependency checking.
-# See these for more info:
-# https://github.com/abseil/abseil-py/issues/79
-# https://hynek.me/articles/conditional-python-dependencies/
-# Environment marker support was added in setuptools 36.2, see
-# https://github.com/pypa/setuptools/blob/master/CHANGES.rst#v3620
-if setuptools_version < (36, 2):
-  if sys.version_info[0:2] < (3, 4):
-    INSTALL_REQUIRES.append('enum34')
-else:
-  # Environment markers are the preferred way: it allows correct non-source
-  # distributions (i.e., wheels) to be generated.
-  INSTALL_REQUIRES.append("enum34; python_version < '3.4'")
-
 _README_PATH = os.path.join(
     os.path.dirname(os.path.realpath(__file__)), 'README.md')
 with open(_README_PATH, 'rb') as fp:
